sys/shell/commands/sc_lwip_netif.c

```diff
@@ -96,7 +149,42 @@ int _lwip_netif_config(int argc, char **argv)
             }
         }
         return 0;
+    } else if (strcmp(argv[1], "help") == 0) {
+        _usage(argv[0]);
+        return 0;
+    } else {
+        LOCK_TCPIP_CORE();
+        struct netif *netif = netif_find(argv[1]);
+        UNLOCK_TCPIP_CORE();
+        if (!netif) {
+            printf("Interface '%s' not found.\n", argv[1]);
+            _usage(argv[0]);
+            return 1;
+        }
+        if (argc == 2) {
+            _netif_list(netif);
+            return 0;
+        } else if (argc == 3) {
+            if (strcmp(argv[2], "up") == 0) {
+                netifapi_netif_set_up(netif);
+                return 0;
+            } else if (strcmp(argv[2], "down") == 0) {
+                netifapi_netif_set_down(netif);
+                return 0;
+            } else if (strcmp(argv[2], "default") == 0) {
+                netifapi_netif_set_default(netif);
+                return 0;
+            }
+        } else if (argc == 4) {
+#ifdef MODULE_LWIP_IPV4
+            if (strcmp(argv[2], "dhcp") == 0) {
+                if (_dhcp_set(netif, argv[3]) == 0) {
+                    return 0;
+                }
+            }
```

Comment:
I think it would be better to move `dhcp` to it's own command.
The GNRC `ifconfig` command is already way too overloaded, let's not repeat that with LWIP